frontend/src/stores/chatStore.ts

```diff
@@ -37,6 +52,55 @@ export const useChatStore = defineStore('chats', {
 			}
 		},
 
+		async createNewChannel(createNewChannelForm: CreateNewChannelForm) {
```

Comment:
Sooooo i was thinking
this is not really functionality of the chatStore 
it just calls an external api to save something, and that api updates the chat store through sockets
so I though it is better fitted in the component itself

I made a branch `Vi/new_channel_suggestion`
didn't change any code, just moved some parts where I think they make more sense, as a suggestion
I feel medium strongly about this, but if you have a good argument why it should be in the store I'm probably easy to convince :P 
If it turns out that the suggestion on my branch messes anything up, we'll just go for it in the store it shouldn't become extra work!

Reply:
Ha i totally agree at first I wanted to work with the chat.service etc to do some checks but I ended up doing it like this because it felt quick and easy, I'll look into your suggestions, for now maybe merge this and then try yours if it doenst fuck things up, merge yours?